valhalla/tyr/serializers.h

```diff
@@ -91,7 +91,7 @@ std::string serializeTransitAvailable(const Api& request,
  * @param results     The vector of trip paths and match results for each match found
  */
 std::string serializeTraceAttributes(
-    const Api& request,
+    Api& request,
```

Comment:
because these serializers now call the new serializer below which modifies the `api` object, the `const` could no longer be used